dpnp/tests/third_party/cupy/statistics_tests/test_histogram.py

```diff
@@ -453,19 +453,18 @@ def test_digitize_nd_bins(self):
                 xp.digitize(x, bins)
 
 
-@pytest.mark.skip("histogramdd() is not implemented yet")
 @testing.parameterize(
     *testing.product(
         {
             "weights": [None, 1, 2],
-            "weights_dtype": [numpy.int32, numpy.float64],
+            "weights_dtype": [numpy.int32, numpy.float32],
```

Comment:
Would it make sense to use default floating type here?
```suggestion
            "weights_dtype": [numpy.int32, cupy.default_float_type()],
```

Reply:
Done